creation/web_base/cvmfs_umount.sh

```diff
@@ -20,10 +20,10 @@
 #       Namratha Urs
 #
 # Version:
-#       1.0
+#       2.0
```

Comment:
Thanks for the explanation. The version info has been removed since, at this point in time, this script is only used internally within the GlideinWMS framework.